app/scripts/directives/notifications/notificationDrawerWrapper.js

```diff
@@ -184,7 +185,7 @@
           // using uid to match API events and have one filed to pass
           // to EventsService for read/cleared, etc
           trackByID: notification.trackByID,
```

Comment:
Would it make sense to keep all the `id` generation/manipulation together on line `180`? I'd tend to think it should always be the same value if possible.

Reply:
Might not be relevant, reviewing again, we have `id`, `trackById`, and `uid`.  Bleh.

Reply:
Either way, I addressed this particular comment. -thanks